<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Launch It!</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="txtIconName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter the desired name of your new Launcher icon here.</string>
    </property>
    <property name="placeholderText">
     <string>Icon Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtComment">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Provide a description of this application.</string>
    </property>
    <property name="placeholderText">
     <string>Icon Description / Comment</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtExecCmd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The terminal command used to start this applicatioin or script</string>
    </property>
    <property name="placeholderText">
     <string>Command to Start App</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtVersionNumber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Application Version Number</string>
    </property>
    <property name="placeholderText">
     <string>Version</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtImagePath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Path to Icon Image</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbTerminal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Is this a Terminal based App?</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>294</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Application</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Directory</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Link</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnFindImage">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbAccessories">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Accessories</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbCode">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Code</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbGraphics">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Graphics</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbFiles">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Files / Folders</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbInfo">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbMusic">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Music</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbHelp">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbPhotos">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>140</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Photos</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbVideos">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Videos</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbWeb">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Web</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbSocial">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Social</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbReference">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Reference</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="grpIcon">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblCategories">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Categories</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>16</width>
      <height>461</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblActions">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Actions</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtActionName">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>240</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Give a right-click action name</string>
    </property>
    <property name="placeholderText">
     <string>Action Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtActionExec">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>240</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Provide the -right-click action command and arguments</string>
    </property>
    <property name="placeholderText">
     <string>Action command</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listActions">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>411</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddaction">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Action</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoveAction">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>500</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Action</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCreateLauncher">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>560</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Launcher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>851</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblMainHeader">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Creating Launcher: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblMainTitle">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>txtIconName</tabstop>
  <tabstop>txtComment</tabstop>
  <tabstop>txtExecCmd</tabstop>
  <tabstop>txtVersionNumber</tabstop>
  <tabstop>txtImagePath</tabstop>
  <tabstop>btnFindImage</tabstop>
  <tabstop>cbTerminal</tabstop>
  <tabstop>comboType</tabstop>
  <tabstop>cbAccessories</tabstop>
  <tabstop>cbCode</tabstop>
  <tabstop>cbFiles</tabstop>
  <tabstop>cbGraphics</tabstop>
  <tabstop>cbHelp</tabstop>
  <tabstop>cbInfo</tabstop>
  <tabstop>cbMusic</tabstop>
  <tabstop>cbPhotos</tabstop>
  <tabstop>cbReference</tabstop>
  <tabstop>cbSocial</tabstop>
  <tabstop>cbVideos</tabstop>
  <tabstop>cbWeb</tabstop>
  <tabstop>txtActionName</tabstop>
  <tabstop>txtActionExec</tabstop>
  <tabstop>btnAddaction</tabstop>
  <tabstop>listActions</tabstop>
  <tabstop>btnRemoveAction</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnCreateLauncher</tabstop>
  <tabstop>grpIcon</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>txtIconName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>lblMainTitle</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>618</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
